openedx/adg/lms/applications/admin.py

```diff
@@ -119,14 +124,59 @@ class WorkExperienceAdmin(admin.ModelAdmin):
     search_fields = ('name_of_organization', 'job_position_title',)
 
 
-class PrerequisiteCourseAdmin(admin.TabularInline):
-    model = PrerequisiteCourse
+class MultilingualCourseInlineFormset(BaseInlineFormSet):
+    """
+    Inline formset for multilingual courses
+    """
+
+    def clean(self):
+        super(MultilingualCourseInlineFormset, self).clean()
+
+        course_group_lang_count = defaultdict(lambda: 0)
+        store = modulestore()
+
+        for form in self.forms:
+            if not form.cleaned_data:
+                continue
+
+            course_info = store.get_course(form.instance.course.id)
+            if not course_info.language:
+                raise ValidationError(_('Please add language of the course from studio.'))
+
+            course_group_lang_count[course_info.language] += 1
+            if course_group_lang_count[course_info.language] > 1:
+                raise ValidationError(
+                    _(
+                        'You cannot add 2 or more courses with same language.'
+                        ' Add courses of different language or change language of a course from studio!'
+                    )
+                )
+
+
+class MultilingualCourseAdmin(admin.TabularInline):
+    """
+    Inline admin for Multilingual Courses
+    """
+
+    model = MultilingualCourse
+    formset = MultilingualCourseInlineFormset
+    MULTILINGUAL_COURSE_INLINE_FORMS_EMPTY_GROUP = 2
+    MULTILINGUAL_COURSE_INLINE_FORMS_NON_EMPTY_GROUP = 0
 
+    def get_extra(self, request, obj=None, **kwargs):
+        """
+        Customized number of inline forms
+        """
+        if obj and obj.multilingual_courses.exists():
+            return self.MULTILINGUAL_COURSE_INLINE_FORMS_NON_EMPTY_GROUP
+        else:
+            return self.MULTILINGUAL_COURSE_INLINE_FORMS_EMPTY_GROUP
 
-@admin.register(PrerequisiteCourseGroup)
-class PrerequisiteCourseGroupAdmin(admin.ModelAdmin):
-    inlines = [PrerequisiteCourseAdmin, ]
-    list_display = ('name', 'prereq_course_count', 'open_prereq_courses_count',)
+
+@admin.register(MultilingualCourseGroup)
+class MultilingualCourseGroupAdmin(admin.ModelAdmin):
+    inlines = (MultilingualCourseAdmin,)
+    list_display = ('name', 'is_prerequisite', 'multilingual_course_count', 'open_multilingual_courses_count')
```

Comment:
Kindly make the choice of trailing commas consistent. Either add them everywhere applicable or remove them from everywhere.

Reply:
In first case there is only 1 element so trailing comma is required to declare a tuple. In second it is not required as we have multiple elements.